storage/simulation/default.script

```diff
@@ -0,0 +1,12 @@
+# operation[-value] first last every repeat
+# not all operations require a value
```

Comment:
heh, here's the overloading of "value". I meant how are "first last every repeat" delimited.